hera_mc/geo_handling.py

```diff
@@ -313,33 +322,292 @@ def get_location(self, to_find, query_date, show_location=False, verbosity='m'):
                     print(L, ' not found.')
         return found_location
 
-    def get_all_everconnected_locations(self):
-        """
-        Returns a list of all of the locations ever connected.
+    def _search_loop(self, stn, at_date, now, m, base_tdelta, full_req):
```

Comment:
I've tried to give a description.  The main problem is the time resolution and trading off speed.  Fortunately now we will not be using this operationally, but are keeping it as a backup.